src/user-lists/entities/contributions-projects.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiModelProperty } from "@nestjs/swagger/dist/decorators/api-model-property.decorator";
+import { Column, Entity } from "typeorm";
+
+@Entity({ name: "user_list_contributors" })
+export class DbContributionsProjects {
+  @ApiModelProperty({
+    description: "The org name of the repo",
+    example: 0,
+    type: "string",
+  })
+  @Column({
+    type: "string",
+    select: false,
+    insert: false,
+  })
+  orgId: string;
```

Comment:
From a JSON consumption standpoint, we use snake case for property names

```suggestion
  org_id: string;
```

Reply:
I'm curious since this in JS/TS land why snake case is preferred? I've typically run into eslint issues with snake case. Obviously, a minor eslint issue, but curious.

Reply:
Generally with APIs producing JSON output, I've seen and used snake case much more than camel case for the JSON output. There shouldn't be many places where we have to access these property names in TS code as we're relying on TypeORM to do the database table -> entity mapping. Its also a more straightforward translation from the database.

Unless there's a way to have it be camel case in the entity and snake case in the output, I'd prefer it to be consistent with existing usage.

Reply:
Sounds good. Forgot about TypeORM.